data/apps.ts

```diff
@@ -45,6 +46,12 @@ export type appsList = {
 }
 export const apps: appsList = {
   android: [
+      {
+      released_on: "May 21, 2023",
+      name: "Focus",
+      icon: focus,
+      url: "https://play.google.com/store/apps/details?id=allen.town.focus.mastodon",
+    },
```

Comment:
Please specify whether this is paid or free (`paid: true|false`)

Reply:
changed , please merge it